VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java

```diff
@@ -10,17 +10,25 @@
 
     String threadId;
     int page = 1;
+    boolean isQuestionType;
+    boolean shouldGetEndorsed;
 
-    public GetCommentListTask(Context context, String threadId, int page) {
+    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
+                              boolean shouldGetEndorsed) {
```

Comment:
> That being said, imo boolean variable names should always begin with helping verbs like is, has, should etc.

"get" is already a verb; there's no need to add another one, I guess :wink:

It's still strange because it doesn't explicitly define that it would get non-endorsed comments if `false` is provided, but I can't think of another good-sounding alternative.